index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html lang="en">
+
```

Comment:
You need an opening html tag

Reply:
![image](https://github.com/aLxMMiu/template-replication-2/assets/11279684/495b9fff-d0e9-489c-9d7d-f3dd14031528)

This happens when doctype is declared, and is followed by the proper html open tag.

![image](https://github.com/aLxMMiu/template-replication-2/assets/11279684/fada26d7-9eef-4d5a-87a2-0d5f7483cd9c)

This is the layout with no doctype tag.

Reply:
Doctype declaration is not the same thing as opening html tag so you can declare both. They are not exclusive of each other https://www.w3schools.com/tags/tag_doctype.asp

Reply:
The tags have been adjusted to be the screengrab, for now.

Reply:
I don't see any of the changes you are highlighting. All looks fine for me locally

Reply:
![image](https://github.com/aLxMMiu/template-replication-2/assets/11279684/01d24afa-d9e8-451b-baed-a581ca0ce993)